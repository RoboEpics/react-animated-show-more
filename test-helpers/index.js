import renderer from 'react-test-renderer';
import Enzyme from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

export const renderSnapshot = (component) => {
  const tree = renderer
    .create(component)
    .toJSON();
  return tree;
};


export const configureEnzyme = () => {
  Enzyme.configure({ adapter: new Adapter() });
};
