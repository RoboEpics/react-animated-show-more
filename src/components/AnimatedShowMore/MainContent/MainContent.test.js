import React from 'react';
import { MainContent } from './index';
import { renderSnapshot } from '../../../../test-helpers';

describe('MainContent', () => {
  describe('Snapshots', () => {
    it('renders', () => {
      const snap = renderSnapshot(
        <MainContent height={200} animationSpeed={500}>
          Hello world
        </MainContent>
      );
      expect(snap).toMatchSnapshot();
    });
  });
});
