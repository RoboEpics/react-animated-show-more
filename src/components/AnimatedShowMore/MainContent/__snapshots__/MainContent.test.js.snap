// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainContent Snapshots renders 1`] = `
<div
  className="AnimatedShowMore__MainContent"
  style={
    Object {
      "maxHeight": "200px",
      "overflow": "hidden",
      "transition": "max-height 500ms linear",
    }
  }
>
  Hello world
</div>
`;
